### PR TITLE
Bug 1864351: Add support for block volumes

### PR DESCRIPTION
**Pull request by yaacov**
`vm-import-operator` introduced a `volumeMode` field in [1] , this PR adds the UI option to use this field.

Screenshots:
Peek 2020-08-25 09-18

[1] https://github.com/kubevirt/vm-import-operator/pull/347

### CODE REVIEW

**Comment by yaacov** [comment]
@suomiy @pcbailey @irosenzw @glekner please review

**Review by suomiy — COMMENTED** [review]
looks good

File: frontend/packages/kubevirt-plugin/src/types/vm-import/ovirt/vm-import.ts [hunk 1]
```diff
@@ -17,11 +17,13 @@ export type NetworkMapping = {
 export type StorageMapping = {
   source: Source; // Storage Domain ID
   target: Target;
+  volumeMode?: 'Block' | 'Filesystem';
 };
 
 export type DiskMapping = {
   source: Source; // Disk ID
   target: Target;
+  volumeMode?: 'Block' | 'Filesystem';
```
[the remark this answers is not in the document]
Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/types/vm-import/ovirt/vm-import.ts]: done :-)

**Pull request merged**